modules/virt-creating-vm-instancetype.adoc

```diff
@@ -58,23 +61,47 @@ Logos of operating systems that are not available in the cluster are shown at th
 +
-Immediately after you install the environment or when the environment is disconnected, the list of volumes to boot from is empty. In that case, three operating system logos are displayed: Windows, {op-system-base}, and Linux. You can add a new volume that meets you requirements by clicking the *Add volume* button.
+Immediately after you install the environment or when the environment is disconnected, the list of volumes to boot from is empty. In that case, three operating system logos are displayed: Windows, {op-system-base}, and Linux. You can add a new volume that meets your requirements by clicking the *Add volume* button.
+ifdef::dynamic-key,virt-create-vms[]
+* Click *Add volume* to upload a new volume or use an existing persistent volume claim (PVC), volume snapshot, or data source. Then click *Save*.
+endif::[]
 
 ifdef::virt-create-vms[]
 . Click an instance type tile and select the resource size appropriate for your workload.
 endif::[]
 ifdef::dynamic-key[]
 . Click the *Red Hat Enterprise Linux 9 VM* tile.
 endif::[]
-. If you have not already added a public SSH key to your project, click the edit icon beside *Authorized SSH key* in the *VirtualMachine details* section.
-. Select one of the following options:
+ifdef::virt-create-vms[]
+. Optional: Choose the virtual machine details, including the VM's name, that apply to the volume you are booting from:
+endif::[]
+ifdef::virt-create-vms[]
+** For a Linux-based volume, follow these steps to configure SSH:
+endif::[]
+
+.. If you have not already added a public SSH key to your project, click the edit icon beside *Authorized SSH key* in the *VirtualMachine details* section.
+.. Select one of the following options:
 
 * *Use existing*: Select a secret from the secrets list.
-* *Add new*:
-.. Browse to the public SSH key file or paste the file in the key field.
-.. Enter the secret name.
-.. Optional: Select *Automatically apply this key to any new VirtualMachine you create in this project*.
+* *Add new*: Follow these steps:
+... Browse to the public SSH key file or paste the file in the key field.
+... Enter the secret name.
+... Optional: Select *Automatically apply this key to any new VirtualMachine you create in this project*.
 .. Click *Save*.
+
+ifdef::virt-create-vms[]
+** For a Windows volume, follow either of these set of steps to configure sysprep options:
+*** If you have not already added sysprep options for the Windows volume, follow these steps:
+... Click the edit icon beside *Sysprep* in the *VirtualMachine details* section.
+... Add the *Autoattend.xml* answer file.
+... Add the *Unattend.xml* answer file.
+... Click *Save*.
+*** If you want to use existing sysprep options for the Windows volume, follow these steps:
+... Click *Attach existing sysprep*.
+... Enter the name of the existing sysprep *Unattend.xml* answer file.
```

Comment:
For the answer file name, if these are referencing actual elements in the web console, then asterisks are fine. But if these are just general references to file names that the user should produce / upload / copy-paste / whichever, then I might still suggest backticks:
```suggestion
... Add the `Autoattend.xml` answer file.
... Add the `Unattend.xml` answer file.
... Click *Save*.
*** If you want to use existing sysprep options for the Windows volume, follow these steps:
... Click *Attach existing sysprep*.
... Enter the name of the existing sysprep `Unattend.xml` answer file.
```

Reply:
They are referencing actual elements in the web console. I wasn't sure how to tag them, so I looked in our CNV docs to see if there are other instances and found that they had been tagged with asterisks.